mobile/test/swift/EngineBuilderTests.swift

```diff
@@ -367,13 +367,13 @@ final class EngineBuilderTests: XCTestCase {
   func testAddingVirtualClustersAddsToConfigurationWhenRunningEnvoy() {
     let expectation = self.expectation(description: "Run called with expected data")
     MockEnvoyEngine.onRunWithConfig = { config, _ in
-      XCTAssertEqual("[test]", config.virtualClusters)
+      XCTAssertEqual(["test"], config.virtualClusters)
       expectation.fulfill()
     }
 
     _ = EngineBuilder()
       .addEngineType(MockEnvoyEngine.self)
-      .addVirtualClusters("[test]")
+      .addVirtualClusters(["test"])
```

Comment:
optional:  you may want to update these to valid yaml while you're in here.  Bonus points for adding a test for multiple virtual clusters to make sure the joining is correct now we know that's a thing

Reply:
When I enable yaml validation for tests I'll be forced to do that there, which I'm doing very shortly.